### ISSUE TITLE
Missing DockStream integration in reinvent_scoring

### ISSUE THREAD

**Issue by iwatobipen** [MolecularAI/Reinvent#34]
Dear Developer,

Thanks for sharing really nice package of AI drug discovery!
To integrate Dockstream with Reinvent 3.0, reinvent_scoring should be modified I think.

* reinvent_scoring/scoring/score_components/score_component_factory.py 

```
# import DockStream from reinvent_scoring.scoring.score_components
from reinvent_scoring.scoring.score_components import DockStream

# And should be added following line  in _deafult_scoring_component_registry.
enum.DOCKSTREAM: DockStream
```

Thanks,

Taka

**Comment by patronov** [comment]
Thank you for using REINVENT and for pointing this issue.
We spotted it recently and it is fixed in the latest version of reinvent-scoring 0.0.51
However there are further breaking changes in that version that are not outlined in the tutorials.
Therefore I gave you access to the private repo `reinvent_scoring` so that you can fix it locally for the time being and install it from your local fix.
There will be another REINVENT release with some breaking changes and more features but we havent prepared the documentation/examples yet.

Best,

Atanas

**Comment by iwatobipen** [comment]
Dear Atanas and Reinvent development team,

I hope you are doing well and thanks again for developing and sharing
really cool products.
The recent version of REINVENT is very useful.
I appreciate your great work.
Thanks,

Taka

2021年10月2日(土) 21:40 patronov ***@***.***>:

**Comment by patronov** [comment]
Thanks for the positive feedback and do let us know if you come across any other issues. 
We value any critical and constructive input.

Best,

Atanas

**Comment by patronov** [comment]
Hi Taka,

We have released REINVENT 3.1 with some new features such as automated curriculum learning and LibINVENT integrated. There are some experimental bits as well and also the support for DockStream is included.

Best,

Atanas

**Comment by iwatobipen** [comment]
Hi Atanas,

Thanks for your notification!
It sounds cool ;)
I'll version up my env and try to use it ASAP.

Kind regards,
Taka
